buildkite_agent.rb

```diff
@@ -0,0 +1,23 @@
+dep "buildkite-agent" do
+  requires "buildkite-agent.managed"
+  requires "buildkite-agent.launchctl"
+end
+
+dep "buildkite-agent.managed", :buildkite_token do
+  requires "homebrew tap".with "buildkite/buildkite"
+
+  met? {
```

Comment:
Is there a reason we need a custom `met?` here? I thought most ".managed" templates handle this automatically...